### PR TITLE
rename dialog's 'message' parameter (part of T702830)

### PR DESCRIPTION
**Pull request by churkin**

### CODE REVIEW

File: js/ui/dialog.js [hunk 1]
```diff
@@ -236,42 +239,36 @@ exports.custom = function(options) {
 /**
  * @name ui.dialogmethods.alert
  * @publicName alert(message,title)
```
Comment (DokaRus) [inline js/ui/dialog.js]:
```suggestion
 * @publicName alert(messageHtml,title)
```

File: js/ui/dialog.js [hunk 2]
```diff
@@ -236,42 +239,36 @@ exports.custom = function(options) {
...
-        };
+exports.alert = function(messageHtml, title, showTitle) {
+    var options = isPlainObject(messageHtml) ? messageHtml : { title, messageHtml, showTitle };
 
     return exports.custom(options).show();
 };
 
 /**
  * @name ui.dialogmethods.confirm
  * @publicName confirm(message,title)
```
Comment (DokaRus) [inline js/ui/dialog.js]:
```suggestion
 * @publicName confirm(messageHtml,title)
```

**Review by DokaRus — COMMENTED** [review]
It's need to rename `publicName` too (see comments above).
LGTM

**Review by DokaRus — APPROVED** [review]

**Review by BingoRUS — APPROVED** [review]
👍

**Review by jsdmitry — APPROVED** [review]

**Review by viterobk — APPROVED** [review]

**Pull request merged**